lib/libospf.h

```diff
@@ -68,9 +68,8 @@
 #define OSPF_ROUTER_PRIORITY_DEFAULT        1
 #define OSPF_RETRANSMIT_INTERVAL_DEFAULT    5
 #define OSPF_TRANSMIT_DELAY_DEFAULT         1
-#define OSPF_DEFAULT_BANDWIDTH		 10000	/* Mbps */
-
-#define OSPF_DEFAULT_REF_BANDWIDTH	100000  /* Mbps */
+#define OSPF_DEFAULT_BANDWIDTH            100 /* Mbps */
+#define OSPF_DEFAULT_REF_BANDWIDTH       1000 /* Mbps */
```

Comment:
Commit 70bd3c4369b1b77 changed the auto-cost reference-bandwidth to 100gbps, and provided a good explanation for that. Why change it to 1gbps now?